packages/edit-site/src/components/layout/index.js

```diff
@@ -155,9 +155,10 @@ function Layout() {
 											isResizableFrameOversized
 										}
 									/>
+									{ areas.mobile }
```

Comment:
TODO: add an inline comment explaining what's going on here. 

`// The mobile sidebar needs access to the SidebarNavigationContext.Provider inside `<SidebarContent />` .